courses/free-courses-ta.md

```diff
@@ -39,12 +40,20 @@
 
 ### Bash and Shell
 
+* [Bash and Scripting Playlist](https://www.youtube.com/playlist?list=PLe61-EtQTlGsMT6fLm0ONaAycUuNK1TeF) - Skill Up With Stan
```

Comment:
```suggestion
* [Linux Shell Scripting - Tamil](https://www.youtube.com/playlist?list=PLe61-EtQTlGsMT6fLm0ONaAycUuNK1TeF) - Skill Up With Stan
```
Don't invent titles! Use the title given by the resource.
All your submission have invented titles. Please go and fix them all.